file/file_util.h

```diff
@@ -51,20 +51,8 @@ extern IOStatus GenerateOneFileChecksum(
     const std::string& requested_checksum_func_name, std::string* file_checksum,
     std::string* file_checksum_func_name,
     size_t verify_checksums_readahead_size, bool allow_mmap_reads,
-    std::shared_ptr<IOTracer>& io_tracer, RateLimiter* rate_limiter = nullptr);
-
-inline IOStatus GenerateOneFileChecksum(
```

Comment:
Is removing this inline function one of those minor fixes that goes along with this PR?